src/javascript/app/pages/trade/common.js

```diff
@@ -1,8 +1,8 @@
 const Defaults         = require('./defaults');
 const Symbols          = require('./symbols');
 const Tick             = require('./tick');
-const contractsElement = require('./contracts.jsx');
-const marketsElement   = require('./markets.jsx');
+const contractsElement = require('./contracts');
```

Comment:
IMO, this reduces the readability as you can't tell the type of contents by just looking at the import statements.

Better to know everyone's opinions.